### ISSUE TITLE
Wallet Connect link seems to be broken in CryptoWallet after upgrade

### ISSUE THREAD

**Issue by Genysys** [brave/brave-browser#12076]
## Description 

After the last upgrade the wallet connect link seems to be broken. As opposed to opening the wallet, it navigates to a blank page. 

## Steps to reproduce 

   1. Go to uniswap https://app.uniswap.org/
   2. Select `Connect Wallet`
   3. Select ` 

## Actual result 
 Navigates to a blank page

## Expected result
Connects to Brave CryptoWallet

## Version/Channel Information:

![Image](https://ibb.co/mDLc6r7)
- Can you reproduce this issue with the current Play Store version? 

N/A
- Can you reproduce this issue with the current Play Store Beta version? 

N/A
- Can you reproduce this issue with the current Play Store Nightly version?

N/A

## Device details
   - Install type (ARM, x86):
   - Device type (Phone, Tablet, Phablet):
   - Android version:
   
### Website problems only
- Does the issue resolve itself when disabling Brave Shields? 
- Does the issue resolve itself when disabling Brave Rewards?
- Is the issue reproducible on the latest version of Chrome?

**Comment by srirambv** [comment]
There is no support for crypto wallet on Android yet or support for extensions.

**Comment by kyleoliveiro** [comment]
This used to work though. It was recently broken.

**Comment by kyleoliveiro** [comment]
To clarify:

Previously, having both Brave and Metamask installed on an Android phone, when clicking on a Wallet Connect link in Brave browser, it would open the Metamask app and prompt for permission to connect to the site.

Currenty, this no longer works. Instead, clicking on a Wallet Connect link now just opens a blank screen as described above by @Genysys.

**Comment by ryanml** [comment]
cc: @ryanbr - for any potential web compat issue here

**Comment by ryanbr** [comment]
Yeah, Seeing an intermittent blank page on load.  Rolled out a possible fix.

https://github.com/easylist/easylist/commit/86b893c4c7f137e5cb63e4881195b355382c07dd

**Comment by bsclifton** [comment]
This issue hasn't had an update in a while - it's met the criteria for being stale. The issue is going to be closed for now. If this is still a problem, please leave a comment or re-open the issue. Thanks!
